analyzers/ResonanceBuilder.py

```diff
@@ -6,7 +6,7 @@
 import pprint 
 import itertools
 
-mass = {23: 91, 25: 125}
+mass = {23: 91, 25: 125, 32:200}
```

Comment:
For resonnances that don't have a fix mass, like 32 (Z') we should be able to overwrite the mass value in the analyser configuration

Reply:
hello, no need to keep reviewing until  i discuss with Valentin, I think we need to go in a different direction. no big deal and I can write the few lines needed, so no worries :-)

Reply:
Yes @clementhelsens, there is now a parameter `nominal_mass`, this dict is just used if it is not present in order not to break any existing user code. But let's see what the results of the discussion with Colin are - @cbernet, I wrote you on skype :-)